sleap/gui/app.py

```diff
@@ -1391,6 +1391,10 @@ def updateStatusMessage(self, message: Optional[str] = None):
             else:
                 self.statusBar().setStyleSheet("color: black")
 
+            # TODO(JS): Add Camera indicator test here
+            if self.state["session"] is not None and current_video is not None:
+                message += f'{spacer}Camera: {self.state["session"].get_camera(video=self.state["video"])}'
```

Comment:
I don't really want to rely on the `Camcorder.__repr__` here. Can we instead print just the `Camcorder.name`?
```suggestion
                camera = self.state["session"].get_camera(video=self.state["video"])
                if camera is not None:
                    message += f'{spacer}Camera: {camera.name}'
```